lib/web3/formatters.js

```diff
@@ -190,11 +192,11 @@ var outputBlockFormatter = function(block) {
  * @returns {Object} log
 */
 var outputLogFormatter = function(log) {
-    if(log.blockNumber !== null)
```

Comment:
These values can also be undefined so that's not a good option either. 
In 1.0 i will use underscore and could use -isUndefined.